raiden/transfer/node.py

```diff
@@ -745,6 +746,14 @@ def handle_receive_unlock(
     return subdispatch_to_paymenttask(chain_state, state_change, secrethash)
 
 
+def handle_update_transport_timestamp(
```

Comment:
nitpick: In raiden_service you call it `sync_token` and here and in other places `transport_timestamp`. I don't mind either notation but it would be adbisable to choose one and stick to it so that few months down the line when we look at the code it will be obvious that the code talks about the same thing.

Looking at the matrix implementation it seems they call it `sync_token` there too.